packages/design-system/src/components/Accordion/Accordion.tsx

```diff
@@ -31,6 +34,10 @@ const handleKeyDown = (e) => {
   }
 };
 
+/**
+ * For information about how and when to use this component,
+ * [refer to its full documentation page](https://design.cms.gov/components/accordion/).
```

Comment:
What do you all think about this generic intro text? Should we word it differently?

Reply:
I like it, I think it should be at the top of the file though.

Reply:
But now that I see you did this for all files, I would not recommend doing the work to change that :)

Reply:
It only seems to work when I put the doc comments right above where the component variable/function is declared